docs/source/algorithms/GetDetectorOffsets-v1.rst

```diff
@@ -18,10 +18,22 @@ defines the offset from the reference spectrum.
 
 The algorithm iterates over each spectrum in the workspace and fits a PeakFunction (default is a
 :ref:`Gaussian <func-Gaussian>` function) to the reference peaks. The fit is used
-to calculate the centre of the fitted peak, and the offset is then
-calculated as:
+to calculate the centre of the fitted peak. Using the fitted peak center,
+there are 3 options for calculating offset, depending on the choice for the "OffsetMode"
+input parameter:
 
-:math:`-peakCentre*step/(dreference+PeakCentre*step)`
+Relative (offset relative to reference position):
+
+:math:`offset = -1*peakCentre*step/(dreference+PeakCentre*step)`
+
+Absolute (offset relative to ideal position):
+
+:math:`offset = -1*peakCentre*step/(dreference+PeakCentre*step) + (dideal - dreference) / dreference`
+
+
+Signed (offset in raw number of bins):
+
+:math:`offset = peak_center*-1`
```

Comment:
This isn't rendering correctly
![2021-06-04-160824_266x42_scrot](https://user-images.githubusercontent.com/5595210/120857360-217f2100-c54f-11eb-92d4-6678b8e770b3.png)